src/course-unit/breadcrumbs/Breadcrumbs.jsx

```diff
@@ -43,14 +49,18 @@ const Breadcrumbs = ({ courseId }) => {
               />
             </Dropdown.Toggle>
             <Dropdown.Menu>
-              {breadcrumbs.section.dropdownItems.map(({ studioUrl, displayName }) => (
-                <Dropdown.Item
-                  href={studioUrl}
-                  className="small"
-                >
-                  {displayName}
-                </Dropdown.Item>
-              ))}
+              {(isLoadingCourseOutline || !breadcrumbs.section.dropdownItems.length)
```

Comment:
if no breadcrumbs, there will be infinite loading. Is it correct?

Reply:
I think, it can't be possible because we must have at least current section or subsection in these dropdowns.